pinns/src/pinns.c

```diff
@@ -69,6 +69,8 @@ int main(int argc, char **argv) {
   };
 
   sysctls = calloc(argc/2, sizeof(char *));
+  if (sysctls == NULL)
```

Comment:
we can use the UNLIKELY macro to let the compiler optimize

Reply:
Done